core/apply/endpoint.go

```diff
@@ -68,3 +97,4 @@ type depTokensResponse struct {
 
 func (r depTokensResponse) error() error { return r.Err }
 func (r blueprintResponse) error() error { return r.Err }
+func (r profileResponse) error() error   { return r.Err }
```

Comment:
the methods should be under the type declaration. let's not continue the trend here.